sdccc/src/main/java/com/draeger/medical/sdccc/configuration/DefaultTestSuiteModule.java

```diff
@@ -50,5 +51,6 @@ protected void configure() {
         bind(Manipulations.class).to(GRpcManipulations.class).in(Singleton.class);
         bind(LocalAddressResolver.class).to(LocalAddressResolverImpl.class).in(Singleton.class);
         bind(ManipulationSerializer.class).to(GsonManipulationSerializer.class).in(Singleton.class);
+        bind(ManipulationLocker.class).asEagerSingleton();
```

Comment:
why is ManipulationLocker binded differently like asEagerSingleton()

Reply:
Because there is no interface to bind it to, it's just the implementation